CHANGELOG.md

```diff
@@ -1,3 +1,103 @@
+### citus v11.2.0 (January 30, 2023) ###
+
+* Adds support for outer joins having a recurring rel in the outer side of the
```

Comment:
recurring rel sounds too technical term, I don't think other than a handful of us could follow that :) Can you try to reword this?